operator/docs/user-guides/open-telemetry.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "OpenTelemetry / OTLP"
+description: ""
+lead: ""
+date: 2024-10-25T12:43:23+02:00
+lastmod: 2024-10-25T12:43:23+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "user-guides"
+weight: 100
+toc: true
+---
+
+## Introduction
+
+Loki 3.0 introduced the OpenTelemetry Protocol (OTLP) as a new API for ingesting log entries in addition to the standard Push API that was available in previous versions of Loki.
+
+Because OTLP is not specifically geared towards Loki but is a standard format, it needs additional configuration on Loki's side to map the OpenTelemetry data format to Loki's data model.
+
+Specifically, OTLP has no concept of "stream labels" or "structured metadata". OTLP instead provides metadata about a log entry in _attributes_ that are grouped into three buckets (resource, scope and log), depending on the number of entries an attribute applies to.
```

Comment:
At the end of this phrase, I'm unsure what you mean: "depending on the number of entries an attribute applies to."

Reply:
I had rewritten this paragraph on the second try. Can you check, if the new text makes this clearer?

Reply:
Yes much better 👍